meson_options.txt

```diff
@@ -54,6 +54,13 @@ option(
   description: 'Whether to build support for KDE System Settings'
 )
 
+option(
+  'config-xdp',
+  type: 'boolean',
+  value: false,
```

Comment:
Why false? Nobody will ever enable it. :)

Reply:
Because we thought it is only used in flatpak runtime and that there is already a manual configuration. Never thought about GNOME OS.

Reply:
Changed it back to true.